.cicd/toolchain-tests.sh

```diff
@@ -15,11 +15,20 @@ else # Linux
 
     ARGS=${ARGS:-"--rm --init -v $(pwd):$MOUNTED_DIR"}
 
-    PRE_COMMANDS="cd $MOUNTED_DIR/build"
-    SET_PYTHON_PATH="export CICD_PYTHON_PATH=/usr/local/bin/python3"
-    TEST="./tools/toolchain-tester/toolchain-tester ../tests/toolchain/"
-
-    COMMANDS="$PRE_COMMANDS && $SET_PYTHON_PATH && $TEST"
+    if [[ "$IMAGE_TAG" == "oraclelinux-8" ]]; then
+        PRE_COMMANDS="cd $MOUNTED_DIR/build"
+        PACKAGE_COMMANDS="yum install -y python38"
```

Comment:
Is it possible to put `yum install -y python38` in the above "oraclelinux-8.dockderfile" (similar with eos repo), so that "oracle-linux-8" don't need to be specially handled here?

Reply:
This is weird.
* https://github.com/EOSIO/auto-eks-buildfleet/blob/master/fleet/automation-eks-eos-tester-fleet.yaml#L69
* https://github.com/EOSIO/auto-eks-buildfleet/blob/master/docker/buildkite-eos-builder.Dockerfile

If you need python3 on the image, then just install python3? `.cicd/platforms/oraclelinux-8.dockerfile`....

These others already do ... https://github.com/EOSIO/eosio.cdt/blob/bf2a6134c3516dc51387587d5b5ffce84c0b2f2f/.cicd/platforms/amazonlinux-2.dockerfile#L6

Reply:
python38 and python3-devel were already in Oracle linux 8 dockerfile,  did you ever have a look into that?

Reply:
> Is it possible to put `yum install -y python38` in the above "oraclelinux-8.dockderfile" (similar with eos repo), so that "oracle-linux-8" don't need to be specially handled here?

As Qing's mention, add a soft link may point to the right place. ln -s blahblah

Reply:
@softprofe  Python3 should already be in the image.  The problem appears to be that CICD_PYTHON_PATH is hardcoded to /usr/local/bin/python3.  But Oracle Linux 8 installs python3 in /usr/bin/python3.8.    A quick fix in the Dockerfile would be to soft link /usr/local/bin/python3 to /usr/bin//python3.8.  Then you don't have to modify .cicd/toolchain-tests.sh.  Personally I think CICD_PYTHON_PATH being hardcoded is a code smell, but a clean fix would require modifying several other Dockerfiles which would be out of the scope of this PR.